src/applications/mhv/secure-messaging/containers/FolderThreadListView.jsx

```diff
@@ -208,17 +202,13 @@ const FolderThreadListView = props => {
       if (threadList?.length === 0) {
         return (
           <>
-            {mhvSecureMessagingBlockedTriageGroup1p0 ? (
+            {!noAssociations &&
```

Comment:
is `!noAssociations` addition here intentional? It doesn't seem that we used it here before, perhaps its a new finding?

Reply:
Yes, this should have been included previously, but it got missed.